app/controllers/contact_controller.rb

```diff
@@ -89,10 +89,14 @@ def confirm_submission
     respond_to do |format|
       format.html do
         hide_report_a_problem_form_in_response
+        params = {}
+        if contact_params["url"] == Plek.new.website_root + contact_govuk_path
+          params[:govuk_contact_form] = true
+        end
         if @contact_provided
-          redirect_to contact_named_contact_thankyou_path
```

Comment:
I'm a bit confused about how this is used. Here we set the query parameter, but where do we read it?